crates/autopilot/src/database/on_settlement_event_updater.rs

```diff
@@ -64,6 +70,15 @@ impl super::Postgres {
             .await
             .context("insert_settlement_observations")?;
 
+            let order_uids = auction_data
+                .order_executions
+                .iter()
+                .map(|(order_uid, _)| *order_uid)
+                .map(Into::into)
```

Comment:
Can be collapsed into one `.map()`.

Reply:
sure! I like it more with two `.maps` because I will put it with one.